### PR TITLE
docs: fix typos

### PR DESCRIPTION
**Pull request by MegaRedHand**
This PR fixes some typos present in the docs, with the help of an online grammar checker.

### CODE REVIEW

File: docs/bitvectors.md [hunk 1]
```diff
@@ -111,22 +111,22 @@ Moving it to little-endian byte order (we go byte by byte and reverse the bits):
...
 
 For this reason, we represent bitvectors in our node as big-endian binaries. That means that we reverse the bytes (a relatively cheap operation) and, for bit addressing, we just use the complementary index. An example:
 
-If we are still representing the number 259 (validators with index 0, 1 and 8 attested) we'll have the two following representations (note, elixir has a `bitstring` type that lets you address bit by bit and store an amount of bits that's not a multiple of 8):
+If we are still representing the number 259 (validators with index 0, 1, and 8 attested) we'll have the two following representations (note, elixir has a `bitstring` type that lets you address bit by bit and store several bits that's not a multiple of 8):
```
Comment (Arkenan) [inline docs/bitvectors.md:122]: I think this one is not correct. What would "several bits that's not a multiple of 8" mean?

File: docs/fork_choice.md [hunk 2]
```diff
@@ -45,7 +45,7 @@ graph LR
 
 Here, individual block weights are represented by "b", while subtree weights are represented by "w". Some observations:
 
-- $W = B$ for all leaf blocks, as leafs are their own whole subtree.
+- $W = B$ for all leaf blocks, as leaves are their whole subtree.
```
Comment (Arkenan) [inline docs/fork_choice.md:48]: You sure about removing this one?

File: docs/fork_choice.md [hunk 3]
```diff
@@ -57,17 +57,17 @@ $$W_N = B_N + \sum_i^{i \in \text{children}[N]}W_i$$
 
 In the previous scheme, there are two rewards:
 
-- Proposer rewards, given to a proposer when their block is included in the chain. This also adds an incentive for them to try to predict the most-likely branch to be the canonical one.
-- Attester rewards, which are smaller. These are given if the blocks they attest to are included.
+- Proposer rewards, are given to a proposer when their block is included in the chain. This also adds an incentive for them to try to predict the most likely branch to be the canonical one.
```
Comment (Arkenan) [inline docs/fork_choice.md:60]: If we introduce "are" here, let's remove the `,` after rewards.

**Review by Arkenan — APPROVED** [review]
Thanks for the typo-fixing! I left a couple of comments/questions, but the PR is approved and can be merged as is.

File: docs/bitvectors.md (hunk 1 above)
Comment (MegaRedHand) [reply docs/bitvectors.md:122]: Yeah, it sounds kind of weird

File: docs/fork_choice.md (hunk 2 above)
Comment (MegaRedHand) [reply docs/fork_choice.md:48]: I think it sounds better like this.

File: docs/fork_choice.md (hunk 3 above)
Comment (MegaRedHand) [reply docs/fork_choice.md:60]: Sure. I removed it in #778

**Pull request merged**